src/Microsoft.ML/Models/OneVersusAll.cs

```diff
@@ -13,8 +13,20 @@ namespace Microsoft.ML.Models
     public sealed partial class OneVersusAll
     {
         /// <summary>
-        /// Create OneVersusAll multiclass trainer.
+        /// One versus all learner (also known as One vs the rest) is a multiclass learner with
+        /// strategy to fit one classifier per class. For each class it trains the provided 
+        /// binary classifier against all the other classes. 
```

Comment:
"One-versus-all, OvA, learner (also known as One-vs.-rest, "OvR") is a multi-class learner with the strategy to fit one binary classifier per class in the dataset.  It trains the provided binary classifier for each class against the other classes, where the current class is treated as the positive labels and examples in other classes are treated as the negative classes."